test/conversations/Conversation.test.ts

```diff
@@ -331,18 +334,17 @@ describe('conversation', () => {
   })
 
   describe('v2', () => {
+    beforeEach(async () => {
+      alice = await newLocalHostClient()
+      bob = await newLocalHostClient()
+      await waitForUserContact(alice, alice)
+      await waitForUserContact(bob, bob)
+    })
+
     it('v2 conversation', async () => {
-      // publish new contact bundles
```

Comment:
This isn't necessary anymore since the correct version of the contact is published by default.